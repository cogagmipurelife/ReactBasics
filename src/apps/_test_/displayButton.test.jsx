import React from "react";
import {create} from "react-test-renderer";
function Button(props){
return <buton> only for testing </buton>;
}
describe("Button", () => {
    test("Matches", ()=> 
    {
        const button =create(<Button/>);
        expect(button.toJSON()).toMatchSnapshot();
    }   );
});