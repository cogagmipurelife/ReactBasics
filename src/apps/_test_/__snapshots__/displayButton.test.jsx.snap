// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Matches 1`] = `
<buton>
   only for testing 
</buton>
`;
